tests/console/validate_vnc_nfs.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that, after remote installation, the VNC connection
+# and the NFS repositories are available for SLE version.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+
+use testapi;
+use lockapi;
+use repo_tools 'validate_repo_properties';
+use version_utils 'is_sle';
+
+sub run {
+    select_console "root-console";
+    if (is_sle) {
```

Comment:
this kind of conditions in modules is what we have tried to avoid since time ago (main.pm issue), so if you separate this in two validations the modules are more clear and reusable and main thing, they are single-purpose. Other advantage is that someone checking in opensuse 'validate_vnc_nfs' would not be confused about why is nfs validation there (which at the end there isn't because in the module there is a condition, which is even worse when in the future someone decide to create a library including the condition -> main.pm history).

I'm not sure if we have something generic to validate repo, if you just split it in two is fine, and it is out of scope to make it generic of course.

Reply:
So, I added separate files for sle and Tumbleweed but due to a bug in Tumbleweed, I have commented out the schedule change. I thought this will make it easy to enable the module, once the bug is fixed. Wdty?

Reply:
Would the opensuse validation module be marked as inactive by travis check, because it is included only in teh schedule comment?

Reply:
normal approach is to create a progress ticket for ourselves to enable it, otherwise is is dead code that someone could enable accidentally.

Reply:
The other issue, which is bigger is that you are repeating the code, so you have the option I mentioned before to split the repo check and the rest or create a library to share the common part.